pcap/batadv_packet.h

```diff
@@ -0,0 +1,631 @@
+/* SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) */
+/* Copyright (C) 2007-2020  B.A.T.M.A.N. contributors:
+ *
+ * Marek Lindner, Simon Wunderlich
+ */
+
+#ifndef _UAPI_LINUX_BATADV_PACKET_H_
+#define _UAPI_LINUX_BATADV_PACKET_H_
+
+#include <asm/byteorder.h>
```

Comment:
None of those three headers are available on all the operating systems on which libpcap operates.  (Note that one of those operating systems is Windows.)

Find a way not to use them.